### ISSUE TITLE
VSCodeInsider_Extension_VisualStudioOnline_keyboard:Keyboard focus is not visible on the "scroll up" button.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#89092]
**Environment Details:​**
VSCode Version: 1.41.0-insider (System setup)
Operating System: MacOS
Mac version: 10.15.1 (19B88)

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Navigate  right panel till "Extension: Visual Studio Online" 
4. Check the keyboard focus is not visible on the "scroll up" button.

**Actual:**
Keyboard focus is not visible on the "scroll up" button.

**Expected:​**
Keyboard focus should  visible on the "scroll up" button.

**​User Impact:​**
Keyboard only user does not understand in which control keyboard focus present.

**Comment by isidorn** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/88792
